src/test-run/commands/utils.js

```diff
@@ -77,3 +78,12 @@ export function isJSExpression (val) {
     return val !== null && typeof val === 'object' && val.type === RAW_API_JS_EXPRESSION_TYPE &&
            typeof val.value === 'string';
 }
+
+export function isClientExecutableCommand (command) {
```

Comment:
If we write it in plain text, it will be "is client-executable command". We make up the word "client-executable", which is not good.

Maybe call it simpler:

```js
export function isExecutableOnClientCommand (command) {
```